### DISCUSSION TITLE
My created account was immediately blocked! Support is silent

### DISCUSSION THREAD

**Discussion by Provizor-071** [community/community#43315] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I created an account at Provizor071@gmail.com just for the sake of authorization in Gitcoin. I have done a lot of work in Gitcoin (here is a video of my work https://youtu.be/87chVv1huFI). On the second day, my Github account was blocked, although I did nothing in it. I wrote to support a week ago - silence.

Can an account be blocked just for registering?

I also heard from my subscribers that they have the same problem.

**Comment by Devcrypto-17** [comment]
Did you ask Gitcoin?
